packages/central-server/scripts/scheduledTask.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+import '@babel/polyfill';
+import { configureEnv } from '../src/configureEnv';
+import { ModelRegistry, TupaiaDatabase } from '@tupaia/database';
+import winston from '../src/log';
+import { TaskOverdueChecker } from '../src/scheduledTasks';
+import * as modelClasses from '../src/database/models';
+
+const TASKS = {
```

Comment:
This is picky but I can just see us getting confused between tasks and schedules tasks. Perhaps we can call these scheduled jobs/events instead so the distinction is clear?

Reply:
yep good point